### ISSUE TITLE
Tick-spacing in scale axis not working properly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rambaut/figtree#89]
```
I have two issues when using the "Automatic scale" option:

a) When you save a tree, the "Tick spacing" value is not saved in the figtree 
block of the nexus file. When opening the tree again in FigTree this box in 
blank and the default value (0.5) is applied. The other options ("Label 
spacing" and "Origin value") seem to work correctly.

b) Most of the times (but not always), when you erase the "Tick space" value in 
order to replace it with your desired value, FigTree freezes.

I'm running FigTree 1.4.2. I have Java 8 update 25 running on Windows 8.1 64 
bits.

I hope this will help you to improve this indispensable tool.

```

Original issue reported on code.google.com by `atanv...@gmail.com` on 9 Feb 2015 at 6:47

**Comment by rambaut** [comment]
Second issue is related to closed Issue #95

**Comment by rambaut** [comment]
First issue now fixed
